tests/test_definitions/test_expectations_v3_api.py

```diff
@@ -91,7 +91,11 @@ def pytest_generate_tests(metafunc):  # noqa C901 - 35
                                 )
                             validator_with_data = datasets[0]
                         else:
-                            if expectation_category == "column_map_expectations":
+                            if expectation_category in [
+                                "column_map_expectations",
+                                "column_pair_map_expectations",
```

Comment:
This enables the JSON-defined Expectations tests for `ColumnPairMapExpectations`